lib/honeydew.ex

```diff
@@ -2,6 +2,11 @@ defmodule Honeydew do
   alias Honeydew.Job
   require Logger
 
+  @type mod_or_mod_args :: module | {module, list}
+  @type node :: atom
+  @type queue_name :: String.t | atom | {:global, String.t | atom}
+  @type supervisor_opts :: Keyword.t
```

Comment:
not `keyword`?

Reply:
Sure. I can change it to `keyword`. It seems weird you can do it either way.

Reply:
Just checked through the docs and `Keyword.t` and `keyword` are identical.

Reply:
i leave the decision to your aesthetics, then :)